src/plugins/intel_gpu/src/graph/graph_optimizer/prepare_primitive_fusing.cpp

```diff
@@ -291,7 +291,7 @@ void prepare_primitive_fusing::fuse_activations(program &p) {
             }
 
             if (use_onednn_impls) {
-                if (input.is_type<reshape>())
+                if (input.is_type<reshape>() || input.is_type<concatenation>())
```

Comment:
Did you check onednn concat fusing test?

Reply:
All TCs are passed with device_id option for dGPU except 'fusings_gpu/conv_fp32_multi_eltwise_concat.basic/0'~2.
These 3 TCs are failed in master branch as well.
It is not related to fused_activation. We may need to resolve through this other PR.